src/ipc-frontend-dbus.c

```diff
@@ -619,9 +622,17 @@ on_name_lost (GDBusConnection *connection,
               const gchar     *name,
               gpointer         user_data)
 {
-    g_info ("on_name_lost: %s", name);
+    g_debug ("%s: %s", __func__, name);
     IpcFrontend *ipc_frontend = IPC_FRONTEND (user_data);
+    IpcFrontendDbus *self = IPC_FRONTEND_DBUS (user_data);
+
+    if (self->connection_state == CONNECTION_STATE_DISCONNECTED) {
+        g_warning ("Failed to acquire DBus name %s. UID %d must be "
```

Comment:
Wonder if this shouldn't be g_critical instead since is more than a warning (the daemon will not start after this)?

Reply:
Reasonable. The daemon will shutdown after this so the code treats the condition as 'critical' since it means there's no reason for the daemon to try to do anything but shutdown if this happens.